File: app/src/main/java/com/example/fatfoxhospital/Nurse.java
package com.example.fatfoxhospital;

public class Nurse {
    private long id;
    private String name;
    private String surname;
    private String user;
    private String password;

    public Nurse(long id, String name, String surname, String user, String password) {
        this.id = id;
        this.name = name;
        this.surname = surname;
        this.user = user;
        this.password = password;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getSurname() {
        return surname;
    }

    public String getUser() {
        return user;
    }

    public String getPassword() {
        return password;
    }
}

File: app/src/main/java/com/example/fatfoxhospital/pr07/Nurse.kt
package com.example.fatfoxhospital.pr07

data class Nurse(
    var id: Long,
    var name: String,
    var surname: String,
    var email: String,
    var user: String,
    var password: String
)

val FAKE_REGISTERED_NURSES = listOf(
    Nurse(
        id = 1L,
        name = "Ana",
        surname = "Gómez",
        email = "ana.gomez@fatfox.com",
        user = "anagomez",
        password = "password1"
    ),
    Nurse(
        id = 2L,
        name = "Borja",
        surname = "Pérez",
        email = "borja.perez@fatfox.com",
        user = "borjap",
        password = "password2"
    )
    // Puedes añadir más ejemplos aquí
)